tests/manual/iot-core/subscribe-device-to-registry-commands.sh

```diff
@@ -1,13 +1,18 @@
-#fixme add asserts that env variables are defined
+#!/bin/sh
 
-#REGISTRY_ID="areiqldr2chj5v0g2rd6"
-#REGISTRY_PASSWORD="zx+z{i1QD6YJ#J#&"
-#DEVICE_ID="arekifdlttq12q6sslle"
-#DEVICE_PASSWORD="HsTvGAN6HFundl1v"
+REGISTRY_ID=$(echo "$IOT_REGISTRY")
+REGISTRY_PASSWORD=$(echo "$IOT_REGISTRY_PASSWORD")
 
+EMPTY_LINE=""
+
+echo "--------------------------------------------------------"
+echo "Credentials: '${REGISTRY_ID}' : '${REGISTRY_PASSWORD}'"
```

Comment:
the same as [here](https://github.com/chaos-adept/iot-dipstick/pull/30/files#diff-d409215c15980a38f6fa6df07f6d55a15e98d30a0c8718f7e5690f43bbe1e926R9)

Reply:
Same here too :D